apps/system/js/notifications.js

```diff
@@ -672,7 +687,8 @@ var NotificationScreen = {
 
   closeNotification: function ns_closeNotification(notificationNode) {
     var notificationId = notificationNode.dataset.notificationId;
-    this.removeNotification(notificationId);
+    var id = notificationNode.id;
+    this.removeNotification(notificationId, id);
```

Comment:
It looks like many of the changes in this file are unnecessary changes to the indentation level. This makes it very hard to review. Please fix that before requesting further review.